cfssl_refresh_cert.py

```
@@ -32,6 +32,9 @@ def refresh_cert_and_key(self):
             "request": self.config["cfssl"]["request"]
         }
 
+        if "profile" in self.config["cfssl"]:
```

Instead of selectively copying fields from the config over do we want to just supply the whole thing? Alternately we could fail on unrecognized properties so that we don't have the current situation where we silently fail

---

This will break the way we currently deploy config.json - I'll have to add a new field for this.